### PR TITLE
[WIP] Add certificate renewal integration test

### PR DESCRIPTION
**Pull request by jsliacan**
Add a cert renewal integration test.

### CODE REVIEW

File: test/integration/utilities_test.go [hunk 1]
```diff
@@ -268,3 +278,28 @@ func (e CodeExitError) Exited() bool {
...
+	retryDuration, err := time.ParseDuration(retryTime)
+	if err != nil {
+		return false, err
+	}
+
+	var contains = false
+
+	for i := 0; i < retryCount; i++ {
+		out, err := exec.Command("bash", "-c", command).Output()
+		outstr := fmt.Sprintf("%s", out)
```
Comment (anjannath) [inline test/integration/utilities_test.go]: use `outstr := string(out)` instead ?

File: test/integration/cert_test.go [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+		})
+
+		It("Prewind clock by 11 months", func() {
+			_, err := exec.Command("bash", "-c", "sudo timedatectl set-ntp off").Output()
+			Expect(err).NotTo(HaveOccurred())
+
+			_, err = exec.Command("bash", "-c", "sudo date -s '11 month'").Output()
+			Expect(err).NotTo(HaveOccurred())
+
+			contains, err := CheckOutputContainsWithRetry(10, "1s", "virsh --readonly -c qemu:///system capabilities", "<capabilities>")
```
Comment (adrianriobo) [inline test/integration/cert_test.go:29]: consider usign common match with retry function at https://github.com/crc-org/crc/blob/main/test/extended/util/util.go#L167

File: test/integration/cert_test.go [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+			Expect(err).NotTo(HaveOccurred())
+			Expect(contains).To(BeTrue())
+		})
+
+		It("start CRC", func() {
```
Comment (adrianriobo) [inline test/integration/cert_test.go:34]: Check crc commands invocation, with https://github.com/crc-org/crc/blob/main/test/extended/crc/cmd/cmd.go

**Comment by jsliacan** [comment]
I suspect this won't be needed. With a 1y rotation, it's hardly an issue if certs fail to renew after such a long time.

**Pull request closed without merging**